File: aoc2015/day02/sol.py
res=0
r2=0
for line in open("input.txt"):
    a,b,c = sorted(map(int, line.strip().split("x")))
    res +=2*(a*b+a*c+b*c)+a*b
    r2 += 2*a+2*b+a*b*c

print(res)
print(r2)

File: aoc2015/day02/sol.rb
data = File.read('input').split("\n").map{|row|row.strip.split("x").map(&:to_i).sort}
p data.map{|l,w,h|3*l*w + 2*w*h + 2*h*l}.sum
p data.map{|l,w,h|2*l + 2*w + l*w*h}.sum
